client/src/app/App.js

```diff
@@ -1751,6 +1752,13 @@ export class App extends PureComponent {
       activeTab
     } = this.state;
 
+    // Allows menu actions to easily pass options
+    if (isArray(action)) {
+      options = action[1];
+      action = action[0];
+    }
+
+    console.log(action, options);
```

Comment:
Get rid of this
```suggestion
    console.log(action, options);
```

Reply:
🤦 

Fixed in [88646ed](https://github.com/camunda/camunda-modeler/pull/4575/commits/88646ed5bfe24254c722d240ab578ad8f98a7015)